pandas/tests/test_style.py

```diff
@@ -0,0 +1,260 @@
+import copy
+from nose import SkipTest
+from unittest import TestCase
+
+import numpy as np
+import pandas as pd
+from pandas import DataFrame
+
+from pandas.core.style import Styler
+import pandas.util.testing as tm
+
+
+class TestStyler(TestCase):
+
+    def setUp(self):
+        np.random.seed(24)
+        self.s = DataFrame({'A': np.random.permutation(range(6))})
+        self.df = DataFrame({'A': [0, 1], 'B': np.random.randn(2)})
+        self.f = lambda x: x
+        self.g = lambda x: x
```

Comment:
test looks like

```python
def test_no_jinja():
    # Separate so it isn't skipped by setUp
    df = pd.DataFrame({'A': [1]})
    if not has_jinja:
        with tm.assertRaises(ImportError):
            Styler(df)
        with tm.assertRaises(ImportError):
            df.style
```

so it always runs, but only really checks when there's not a jinja. Know of a better way to handle this. I did some brief searches and didn't find much for testing for ImportErrors, and I don't really want to much with the import machinery and caches :)